src/ui/autocomplete.ts

```diff
@@ -511,6 +548,10 @@ class AutocompleteUI {
     return this;
   }
 
+  public generateUUID = (): string => {
+    return crypto.randomUUID();
```

Comment:
I wanted to use crypto.randomUUID() but not sure if this is the case but maybe:

https://github.com/parcel-bundler/parcel/issues/8820